recipes/exprtk/all/conandata.yml

```diff
@@ -1,4 +1,6 @@
 sources:
+  "latest":
+    url: "https://www.partow.net/downloads/exprtk.zip"
```

Comment:
Unfortunately - if a library doesn't provide sources that are sha/content stable, it is not eligible for inclusion in Conan Center Index - as it does not allow for content validation/verification. This is particularly a problem when the code comes some a self-hosted website, rather than a well known source (e.g. github, etc).

Please do reach out to the author in case this is something they are willing to address - otherwise, I'd imagine the only alternative accept is a github mirror that republishes `0.0.2` in a way that the sha is stable, and the contents are verifiably the same as https://github.com/ArashPartow/exprtk/archive/0.0.2.tar.gz.

(We do have it as a task to address the only other case of this in Conan Center, which is the msys2 recipe)

Reply:
I've been in contact with the author, they will be making a new version at some point. When they do I will update the recipe.